api.planx.uk/gis/classifiedRoads.js

```diff
@@ -0,0 +1,73 @@
+import "isomorphic-fetch";
+
+// Passport key comes from Digital Planning Schemas googlesheet
+export const PASSPORT_FN = "road.classified";
+
+export const classifiedRoadsSearch = async (req, res, next) => {
+  try {
+    if (!req.query.geom)
+      return next({ status: 401, message: "Missing required query param `?geom=`" });
+
+    // Create an OGC XML filter parameter value which will select the road features (lines)
+    //   that intersect with the buffered site boundary (polygon) coordinates
+    //   ref https://labs.os.uk/public/os-data-hub-examples/os-features-api/wfs-example-intersects#maplibre-gl-js
+    const xml = `
```

Comment:
OpenLayers has some nice APIs for reading and writing OGC Filters. This would handle little quirks like the x, y -> y, x flip.

 I think this is a simple enough example to have written as a string, but something to be aware of 👍

Reply:
Yeah, I am very much trying to push off introducing openlayers as a dependency here until we really have a strong case for it - but definitely aware of this if our filters become too complex to easily handle/understand as string templates!